Jenkinsfile

```diff
@@ -14,6 +14,12 @@ pipeline {
       description: 'Determine if the app requires authentication.',
       defaultValue: true
     )
+
+    stringParam(
+      name: 'AWS_IAM_ROLE',
+      description: 'IAM role that the webapp will assume (optional)',
+      defaultValue: '',
```

Comment:
I think the trailing comma might cause a problem?

Reply:
We already use a trailing comma in the `IP_RESTRICTIONS` param and it's not causing problems.